test/Microsoft.AspNet.FileProviders.Physical.Tests/PhysicalFileProviderTests.cs

```diff
@@ -20,14 +20,16 @@ public class PhysicalFileProviderTests
         [Fact]
         public void ExistingFilesReturnTrue()
         {
-            var provider = new PhysicalFileProvider(Directory.GetCurrentDirectory());
-            var info = provider.GetFileInfo("File.txt");
-            Assert.NotNull(info);
-            Assert.True(info.Exists);
-
-            info = provider.GetFileInfo("/File.txt");
-            Assert.NotNull(info);
-            Assert.True(info.Exists);
+            using (var provider = new PhysicalFileProvider(Directory.GetCurrentDirectory()))
```

Comment:
Seems like all the changes from here on are about reducing file system burden while testing i.e. releasing resources ASAP. If there's an exception to that, let me know.